### DISCUSSION TITLE
Versions on GitHub don't match GitHub releases

### DISCUSSION THREAD

**Discussion by iainbeeston** [avo-hq/avo#1099] [General]
I've just noticed that the latest release on ruby gems is v2.11.1 but that version doesn't exist as a tag or a release on GitHub. Is that expected? (It could be a sign that someone has hacked the avo rubygems account, or maybe the latest tags haven't been pushed?)

Here's ruby gems:

![Screenshot 2022-07-26 at 09 39 53](https://user-images.githubusercontent.com/111963/180963063-54f0cfa1-40e4-4726-9a43-9495974e2add.jpg)

Here's GitHub:

![Screenshot 2022-07-26 at 09 40 24](https://user-images.githubusercontent.com/111963/180963194-30bba7c4-2fc4-4295-9da2-b74fba14b225.jpg)

![Screenshot 2022-07-26 at 09 40 11](https://user-images.githubusercontent.com/111963/180963206-d59777b0-d324-427b-8908-5ba47185c411.jpg)

**Comment by adrianthedev** [comment]
I detailed this a bit in our [release schedule document](https://github.com/avo-hq/avo/blob/main/RELEASE.MD). The versions that don't appear in the GH releases page are manually released by us when we want to quickly push out a bugfix.
